src/common/mmo.h

```diff
@@ -838,6 +838,13 @@ enum e_pc_reg_loading {
 	PRL_ALL = 0xFF,
 };
 
+typedef enum PARTY_MEMBER_WITHDRAW {
+	PARTY_MEMBER_WITHDRAW_LEAVE,	  ///< /leave
+	PARTY_MEMBER_WITHDRAW_EXPEL,	  ///< Kicked
+	PARTY_MEMBER_WITHDRAW_CANT_LEAVE, ///< TODO: Cannot /leave
+	PARTY_MEMBER_WITHDRAW_CANT_EXPEL, ///< TODO: Cannot be kicked
```

Comment:
Could that be the correct return for clif_parse_RemovePartyMember when you are on a party locked map?
While I looked this up: Should we not check the target player's map too?